hles/transformation/src/main/scala/org/broadinstitute/monster/dap/environment/WalkabilityTransformations.scala

```diff
@@ -0,0 +1,34 @@
+package org.broadinstitute.monster.dap.environment
+
+import org.broadinstitute.monster.dap.RawRecord
+import org.broadinstitute.monster.dogaging.jadeschema.fragment.EnvironmentWalkability
+
+object WalkabilityTransformations {
+
+  /**
+    * Parse all walkability variables out of a raw RedCap record,
+    * injecting them into a partially-modeled environment record.
+    */
+  def mapWalkabilityVariables(rawRecord: RawRecord): EnvironmentWalkability = {
+    var walkscore: Option[Double] = None
+    var walkscoreDate: Option[String] = None
+
+    if (!rawRecord.getOptional("wv_walkscore").contains("NA")) {
+      walkscore = rawRecord.getOptional("wv_walkscore").map(_.toDouble)
+    }
+    if (rawRecord.getArray("wv_walkscore_date").nonEmpty) {
```

Comment:
Can you add a comment describing why this nested conditional exists (I think I remember from our pairing, but we will for sure forget a month from now).

Reply:
We observed instances of "NA" in data returned from RedCap - thanks I will add the comment so we don't scratch our heads in the future

Reply:
could we include handling for `"NA"` values in our various `getOptional/getArray/getWhatever` methods rather than explicitly handling that here, or would that potentially swallow cases where `"NA"` is a valid value for a field?